app/server/utils/ussd/kenya_ussd_processor.py

```diff
@@ -23,7 +23,7 @@ def process_request(session_id: str, user_input: str, user: User) -> UssdMenu:
                 return UssdMenu.find_by_name(new_state)
         # new session
         else:
-            if user.is_resetting():
+            if user.is_resetting_pin():
```

Comment:
hmm i don't think we need reset tokens for pin, since we don't ever use them to verify anything. all we ever need is to check if we're resetting the pin or not. this could just be done via a boolean that we set and unset. is there any situation we'd need a token for pin?

Reply:
I can imagine there's a good chance we'd need it for the android app in the future, though:
- right now there's no way to reset a forgotten pin from the android phone. 
- I think normally the flow for reseting a pin on a flow is different from setting a password - for pin u normally just get a new one-time code via sms or something right?
@PhilipWafula was this what u meant?

Reply:
one thing i'm not clear on @PhilipWafula @wangfowen is how pin reset flow is supposed to work on the USSD interface - is it something the user is able to do at all, or is an admin required to do something? I couldn't find any use of `change_current_pin` in the code (though i'm guessing that might just be because it's a TODO?)

Reply:
there's a ticket for adding that to admin interface. right now it's not doable